### DISCUSSION TITLE
Images Missing After Upgrade from 0.79.1 to 0.80.2

### DISCUSSION THREAD

**Discussion by rizkihidayatnet** [outline/outline#7666] [Self Hosting]
Hi, I upgraded from 0.79.1 to 0.80.2, and after the successful update, all my images are gone.
I'm using Easypanel with the Docker option to perform the upgrade, and the images were stored locally.

Is there an option to keep the files during the upgrade process?

**Comment by tommoor** [comment]
Nothing about upgrading deletes your files, it sounds like the Docker volume wasn't persisted perhaps – totally depends on your docker setup.
